src/ontology/imports/foodon_product_import.owl

```diff
@@ -62190,6 +62196,12 @@ soy grits have a particle size larger than 100 mesh.&quot; [https://link.springe
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_03303557">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001695"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/RO_0001000"/>
+                <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/FOODON_03302091"/>
```

Comment:
Probably better to say this derives from some bos taurus.  Saying it derives from beef broth seems a bit circular.

Reply:
actually just take this out, and leave it be handled by parent class.

Reply:
Right, 'beef broth was added as comment section also support this statement by saying that it * derives from 'broth or stock'. But I agree that it seems a bit circular logic.
Secondly, it has a parent class 'cattle broth or stock food product', which itself should 'derives from' some 'bovinae'. Thus I think'beef' in 'beef gravy' can only be defined if we add axiom in this class ('derives from' some 'Bos taurus'). OBecause otherwise it cannot be handled by its parent class.
Same goes for 'beef gravy mix'.

Reply:
Okay I've checked. 'Bos taurus', 'Bos primigenius taurus', 'cattle' are all synonyms as per NCBI.
So yes we can add the axiom in parent term. I have updated 'cattle broth or stock food product' and 'cattle cured meat food product'.